CONTRIBUTING.md

```diff
@@ -165,4 +166,18 @@ Please keep in mind that the team may **suspend or reject** pull requests. There
 -   Poor quality code.
 -   Other [development priorities](https://testcafe.io/402949/roadmap) may take precedence.
 
-Please also note that we do not merge pull requests until the changes are documented.
+We merge pull requests after the changes are documented. If you want to document a new capability, write a comment with your description to a pull request. Please, follow the [guidelines](#guidelines) during the writing. 
```

Comment:
* you cannot "write to" a pull request
* a pull request -> *the* pull request because we're talking about specific new capabilities
* the third sentence includes a gerund and an unnecessary introductory word.
```suggestion
We merge pull requests after the changes are documented. If you want to document a new capability, add a comment with the description to the pull request. Review our [writing guidelines](#guidelines) before you proceed.
```